tests/conftest.py

```diff
@@ -74,6 +74,7 @@
 
 def make_data_file(i, run=None, timeset=None):
     return DataFile(
+        id=i,
```

Comment:
I'm surprised it worked without this. The calls to this function include this argument!